api/porch/types_package.go

```diff
@@ -21,36 +21,36 @@ import (
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// Package
+// PorchPkg
 // +k8s:openapi-gen=true
-type Package struct {
+type PorchPkg struct {
```

Comment:
This needs to change for sure, and it's not a big thing to break this, based on https://github.com/nephio-project/nephio/discussions/450 it was never fully implemented. But for other CRs, is the name change really needed?

Reply:
If the CR naming can be made consistent in the code (eg using PorchPkg everywhere instead of Package) *without changing the json/yaml resource files* I'd prefer to have them consistent, but not otherwise. I have some more work to do here to check everything 100%